test/suites/edgexfoundry/network_test.go

```diff
@@ -10,22 +10,7 @@ func TestNetworkInterface(t *testing.T) {
 		utils.SnapStop(t, coreMetadataService)
 	})
 
-	// check network interface status for core-metadata service
 	utils.SnapStart(t, coreMetadataService)
-
-	t.Run("listen default port "+coreMetadataDefaultServicePort, func(t *testing.T) {
-		utils.WaitServiceOnline(t, 60, coreMetadataDefaultServicePort)
-	})
-
-	t.Run("not listen on all interfaces", func(t *testing.T) {
-		utils.RequireListenAllInterfaces(t, false, coreMetadataDefaultServicePort)
-	})
-
-	t.Run("listen localhost", func(t *testing.T) {
-		utils.RequireListenLoopback(t, coreMetadataDefaultServicePort)
-		utils.RequirePortOpen(t, coreMetadataDefaultServicePort)
-	})
-
```

Comment:
Any reason that the remaining add/port tests can't be passed to the utils TestCommon?

Reply:
I am not sure if this is a common test for other test suites.  I will do another device service to check.

Reply:
This entire network test has been removed now.

Reply:
This entire network test has been removed now.